public/scripts/editor/js/bramble-editor.js

```diff
@@ -4,15 +4,17 @@ define(function(require) {
       ProjectRenameUtility = require("fc/project-rename"),
       FileSystemSync = require("fc/filesystem-sync"),
       SyncState = require("fc/sync-state"),
-      Project = require("project");
+      Project = require("project"),
+      SSOOverride = require("sso-override");
 
   return {
     create: function(options) {
-      var makeDetails = options.makeDetails;
+      var project = options.project;
```

Comment:
I think it's confusing to call the data `project`, since we also have `Project`.  Can you name this `projectDetails` or something, so that it's clear what's what?